website/docs/docs/core/connect-data-platform/fabric-setup.md

```diff
@@ -332,7 +332,7 @@ You can optionally set the principal who should own all schemas created by dbt.
 CREATE SCHEMA [schema_name] AUTHORIZATION [schema_authorization]
 ```
 
-A common use case is to use this when you are authenticating with a principal who has permissions based on a group, such as an AAD group. When that principal creates a schema, the server will first try to create an individual login for this principal and then link the schema to that principal. If you would be using Azure AD in this case,
+A common use case is to use this when you are authenticating with a principal who has permissions based on a group, such as a Microsoft Entra ID group. When that principal creates a schema, the server will first try to create an individual login for this principal and then link the schema to that principal. If you would be using Microsoft Entra ID in this case,
 then this would fail since Azure SQL can't create logins for individuals part of an AD group automatically.
```

Comment:
not sure if im confused but does AD mean active directory here?
```suggestion
then this would fail since Azure SQL can't create logins for individuals part of an AD group automatically.
```